charts/vector/README.md

```diff
@@ -32,15 +32,12 @@ helm install --name <RELEASE_NAME> \
 
 ### Upgrading
 
-#### From vector-agent
+#### From original charts
 
```

Comment:
Aha, just saw this while looking to see if there was an issue/PR filed for https://github.com/vectordotdev/helm-charts/issues/77

I think it'd be super helpful to add a note similar to https://github.com/vectordotdev/helm-charts/pull/76 (feel free to close that) for a little extra context for folks!

```suggestion

The [Vector chart](charts/vector/README.md) is the recommended way to deploy Vector in either the Aggregator or Agent roles. The original charts are deprecated, please see the following migration guides: 
```

Reply:
👍  I wanted to make sure we had the new `vector` chart in a stable (bug free 😅 ) spot before messaging it as a replacement of the original charts. I suppose once the migration docs are in place and this is merged that'll probably be a good time to do so.

As a side-note your suggested change should _technically_ go into `README.md.gotmpl` otherwise `helm-docs` would overwrite it 😃